copernicusmarine/download_functions/subset_xarray.py

```diff
@@ -511,6 +511,25 @@ def _update_dataset_coordinate_attributes(
                     coordinate_attributes.remove("units")
                 elif coordinate_label in ["depth", "elevation"]:
                     coordinate_attributes.append("positive")
+                if (
+                    coord.attrs == {}
+                ):  # specific dataset case without lon and lat attrs
```

Comment:
If you want to add a comment remember to comment the "why" and not the "what" because reading the code we see that it is a case where the lat lon are don't have attributs

I would put something like: 
```suggestion
                    coordinate_attributes.append("positive")
                # TODO: delete this when fixed on ARCO processor side
                # example dataset: klshdflksjdf
                if (
                    coord.attrs == {}
                ): 
```